completion_aggregator/models.py

```diff
@@ -299,3 +299,15 @@ def __str__(self):
         if self.resolved:
             parts.append('*')
         return ''.join(parts)
+
+
+@python_2_unicode_compatible
+class CacheGroupInvalidation(models.Model):
```

Comment:
Couldn't this be stored in the cache as well? Since the invalidation entry for a cache group will always happen after the entry itself, the original entry will also expire before the invalidation expires, so I don't think this will lead to data not being invalidated when it should. Or am I missing something about caching?

Reply:
I'm not positive whether edx uses redis or memcache for caching, but if redis, there could be multiple cache nodes, and if one of them goes down, some data might get lost.  It might be okay, but I don't feel confident that the invalidation will always be present after the entry expires.  I'd rather have a durable database entry, just to be on the safe side.

Reply:
Sure, that makes sense.